docs/topics/logging.rst

```diff
@@ -83,10 +82,10 @@ path::
 
 .. seealso::
 
-    Module logging, `HowTo <https://docs.python.org/2/howto/logging.html>`_
+    Module logging, `HowTo <https://docs.python.org/3/howto/logging.html>`_
```

Comment:
InterSphinx can be used here as well.

Selecting *Show source* on https://github.com/python/cpython/blob/3.8/Doc/howto/logging.rst links to https://github.com/python/cpython/blob/3.8/Doc/howto/logging.rst, where you can select *Raw* to see the plain text content of that page. I think it may be possible to link to it with something like `` :doc:`python:howto/logging` `` (it may need testing), or use the reference tag of the basic howto: `` :ref:`logging-basic-tutorial` ``.

Reply:
Thanks for the suggestion. I tried `` :doc:`python:howto/logging` `` and it works. I ran 
```bash
python -msphinx.ext.intersphinx https://docs.python.org/3/objects.inv > objects.txt
```
to get a list of all the links in python.